sdk/cosmos/azure-cosmos-spark_3-3_2-12/CHANGELOG.md

```diff
@@ -1,16 +1,10 @@
 ## Release History
 
-### 4.38.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
+### 4.38.0 (2025-05-13)
 
 #### Bugs Fixed
```

Comment:
I guess I shuld have also inlcuded the CosmosPagedIterable fix here

Reply:
Fixed